odoo/fields.py

```diff
@@ -2376,6 +2376,32 @@ def __init__(self, comodel_name=Default, relation=Default, column1=Default,
             **kwargs
         )
 
+    def _get_mro_has_field_relation(self, model, name):
+        for mro in type(model).__mro__:
+            if hasattr(mro, '_abstract') and\
+                    not mro._abstract and name in mro.__dict__:
+                field = mro.__dict__[name]
+                return self.relation == field.args.get('relation')
```

Comment:
I am not sure about this. 'relation' is in `field` instead if `field.args` (at least in v9) and also it gives me an error of maximum recursion depth exceeded

Reply:
I think you have tested relation value before that line of code. Fields properties are instantiated after setup_attrs() function

Reply:
but if the property is not instantiated yet, why you check for that property?

Reply:
At that point (we are after setup_attrs()), self.relation is instantiated. But the base field maybe not. So, that's why we need to get the base field args value instead.

Reply:
Fine!